src/frontend/js/utils/AbilitiesHelper/types.ts

```diff
@@ -0,0 +1,14 @@
+import { CourseLight } from 'types/Joanie';
+import { JoanieUserProfile } from 'types/User';
+
+export enum JoanieUserActions {
+  CAN_ACCESS_TEACHER_DASHBOARD = 'can_access_teacher_dahboard',
+}
+export const isJoanieUserEntity = (entity: Entity): entity is JoanieUserProfile =>
+  JSON.stringify(Object.keys(entity as JoanieUserProfile).sort()) ===
+  JSON.stringify(['abilities', 'full_name', 'id', 'is_staff', 'is_superuser', 'username']);
```

Comment:
We tend to ban the use of arrow function without explicit return statement mainly to ease function refactoring and readability.